spaceship/lib/spaceship/connect_api/models/custom_app_organization.rb

```diff
@@ -0,0 +1,43 @@
+require_relative '../model'
+module Spaceship
+  class ConnectAPI
+    class CustomAppOrganization
+      include Spaceship::ConnectAPI::Model
+
+      attr_accessor :device_enrollment_program_id
```

Comment:
consider more consistent names for these attributes? `dep_id` and `dep_name` maybe?

Reply:
This is the name of it in the App Store Connect API model/resource so we've been sticking with those exact same names so it also matches what is (or will be) in the official doc 😇